mmdet3d/models/detectors/sassd.py

```diff
@@ -0,0 +1,135 @@
+import torch
+from mmcv.ops import Voxelization
+from mmcv.runner import force_fp32
+from torch.nn import functional as F
+
+from mmdet3d.core import bbox3d2result, merge_aug_bboxes_3d
+from mmdet.models import DETECTORS
```

Comment:
Please use `mmdet3d.models import DETECTORS` instead for future registry refactor compatibilty.  More details can refer to #1412